pkg/sfu/buffer/rtpstats.go

```diff
@@ -792,6 +829,8 @@ func (r *RTPStats) SetRtcpSenderReportData(srData *RTCPSenderReportData) {
 	srDataCopy := *srData
 	srDataCopy.RTPTimestampExt = uint64(srDataCopy.RTPTimestamp) + cycles
 
+	r.maybeAdjustFirstPacketTime(srDataCopy.RTPTimestampExt)
```

Comment:
This is not strictly needed as this is for upstream and we do not use the `firstTime` of upstream in any useful way. But, doing this to be consistent and also to see if this happens a lot.